### PR TITLE
- Converted strings from VisitorBuilder into enum

### PR DESCRIPTION
**Pull request by damianszczepanik**
- Added validation for score strategy
- Fixed EOL in TestFile.java so test does not fail on Windows

### CODE REVIEW

File: src/main/java/pl/touk/sputnik/connector/ConnectorDetails.java [hunk 1]
```diff
@@ -36,6 +36,10 @@ private void validate() {
         notBlank(host, "You must provide non blank connector host");
         notBlank(username, "You must provide non blank connector username");
         notBlank(password, "You must provide non blank connector password");
+
+        String strategy = ConfigurationHolder.instance().getProperty(GeneralOption.SCORE_STRATEGY);
+        // IllegalArgumentException is thrown when strategy could not be mapped to enum
+        ScoreStrategies.valueOf(strategy);
```
Comment (pjagielski) [inline src/main/java/pl/touk/sputnik/connector/ConnectorDetails.java]: maybe extract method instead inline comment

```java
private void validateScoreStrategy throws IllegalArgumentException {
    ScoreStrategies.valueOf(strategy);
}
```

**Comment by SpOOnman** [comment]
What about this PR?

**Comment by SpOOnman** [comment]
@damianszczepanik is this PR still valid or outdated?

**Comment by SpOOnman** [comment]
I close it as outdated.

**Pull request closed without merging**